Common/OpenCL/ITKimprovements/itkGPUImage.h

```diff
@@ -96,6 +96,17 @@ class ITK_TEMPLATE_EXPORT ITKOpenCL_EXPORT GPUImage : public Image<TPixel, VImag
 
   using NeighborhoodAccessorFunctorType = NeighborhoodAccessorFunctor<Self>;
 
+  /** Override Rebind and RebindImageType of itk::Image class */
+  template <typename UPixelType, unsigned int VUImageDimension = VImageDimension>
+  struct Rebind
+  {
+    using Type = itk::GPUImage<UPixelType, VUImageDimension>;
+  };
+
+  template <typename UPixelType, unsigned int VUImageDimension = VImageDimension>
+  using RebindImageType = itk::GPUImage<UPixelType, VUImageDimension>;
```

Comment:
Note to ourselves: remember to consider to propose a similar specialization for ITK's GPU image!